diff --git a/dist/index.js b/dist/index.js
index 6e0a40d99edf0a3e2ed405455c25bed6a48c1c5a..599a74aa6fe5736e734a6225b3a1d5f36b05313a 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -144,9 +144,6 @@ var ScrollAreaViewport = React2.forwardRef(
       /* @__PURE__ */ (0, import_jsx_runtime.jsx)(
         "style",
         {
-          dangerouslySetInnerHTML: {
-            __html: `[data-radix-scroll-area-viewport]{scrollbar-width:none;-ms-overflow-style:none;-webkit-overflow-scrolling:touch;}[data-radix-scroll-area-viewport]::-webkit-scrollbar{display:none}`
-          },
           nonce
         }
       ),
diff --git a/dist/index.mjs b/dist/index.mjs
index fac521d1e293331d6c358170552a0d15ebbc04c6..5d99058c4ecc08415665b23f635793ceeab99d81 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -99,9 +99,6 @@ var ScrollAreaViewport = React2.forwardRef(
       /* @__PURE__ */ jsx(
         "style",
         {
-          dangerouslySetInnerHTML: {
-            __html: `[data-radix-scroll-area-viewport]{scrollbar-width:none;-ms-overflow-style:none;-webkit-overflow-scrolling:touch;}[data-radix-scroll-area-viewport]::-webkit-scrollbar{display:none}`
-          },
           nonce
         }
       ),
